F1_Produtos e Serviços/get-personal-credit-cards/get-personal-credit-cards-7.1.json

```diff
@@ -0,0 +1,1111 @@
+                                        "maximumRate": "0.9000"
+                                    }
+                                ],
+                                "instalmentRates": [
+                                    {
+                                        "referentialRateIndexer": "PRE_FIXADO",
+                                        "rate": "NA",
+                                        "applications": [
+                                            {
+                                                "interval": "1_FAIXA",
+                                                "indexer": {
+                                                    "rate": "0.0987"
+                                                },
+                                                "customers": {
+                                                    "rate": "0.1500"
+                                                }
+                                            },
+                                            {
+                                                "interval": "2_FAIXA",
+                                                "indexer": {
+                                                    "rate": "0.1600"
+                                                },
+                                                "customers": {
+                                                    "rate": "0.3500"
+                                                }
+                                            },
+                                            {
+                                                "interval": "3_FAIXA",
+                                                "indexer": {
+                                                    "rate": "0.3600"
+                                                },
+                                                "customers": {
+                                                    "rate": "0.2000"
+                                                }
+                                            },
+                                            {
+                                                "interval": "4_FAIXA",
+                                                "indexer": {
+                                                    "rate": "0.5890"
+                                                },
+                                                "customers": {
+                                                    "rate": "0.3000"
+                                                }
+                                            }
+                                        ],
+                                        "minimumRate": "0.0456",
+                                        "maximumRate": "0.0865"
+                                    }
+                                ],
+                                "otherCredits": [
+                                    {
+                                        "code": "SAQUE_A_CREDITO"
```

Comment:
Campo additionalInfo é obrigatorio. Verificar em tds as ocorrencias

Reply:
De acordo com a documentação o campo aditionalInfo é obrigatório se o código for OUTROS.
![image](https://user-images.githubusercontent.com/22627896/127517270-9139f6b8-4990-4c5d-a0fa-0be3ea1a7f12.png)